test/remoting.integration.js

```diff
@@ -316,6 +331,12 @@ function getFormattedMethodsExcludingRelations(methods) {
   });
 }
 
+function getCreateMethod(methods) {
+  return methods.filter(function(m) {
```

Comment:
I find it confusing that a function called `getCreateMethod` returns an array of instances. Please use `methods.find` instead of `methods.filter` - see https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find